account_invoice_overdue_reminder/models/account_invoice.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    no_overdue_reminder = fields.Boolean(
+        string='Disable Overdue Reminder',
+        track_visibility='onchange')
+    overdue_reminder_ids = fields.One2many(
+        'account.invoice.overdue.reminder',
+        'invoice_id',
+        string='Overdue Reminder Action History')
+    overdue_reminder_last_date = fields.Date(
+        compute='_compute_overdue_reminder',
+        string='Last Overdue Reminder Date', store=True)
+    overdue_reminder_counter = fields.Integer(
+        string='Overdue Reminder Count', store=True,
+        compute='_compute_overdue_reminder',
+        help="This counter is not increased in case of phone reminder.")
+    overdue = fields.Boolean(compute='_compute_overdue')
+
+    _sql_constraints = [(
+        'counter_positive',
+        'CHECK(overdue_reminder_counter >= 0)',
+        'Overdue Invoice Counter must always be positive')]
+
+    @api.depends('type', 'state', 'date_due')
+    def _compute_overdue(self):
+        today = fields.Date.context_today(self)
+        for inv in self:
```

Comment:
```suggestion
        for inv in self.filtered('date_due'):
```
Because `inv.date_due < today` will fail if `date_due` is False

Reply:
I wasn't aware that date_due could be False on an open invoice. When does it happen ?

Reply:
When `property_payment_term_id` is not set on a partner, it is not filled when a new invoice in created so no `date_due`.

Reply:
I'm sorry, but that's not true. When the payment term is empty, date_due = date_invoice. Here is the relevant code in the account module: https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L1287

Reply:
Indeed you are right, the problem comes from open invoices created in OpenERP 6.1 before the migration.
Sorry for the buzz.